quickstart/101-attestation-provider/main.tf

```diff
@@ -1,22 +1,10 @@
-terraform {
-
-  required_version = ">=0.12"
-    
-  required_providers {
-    azurerm = {
-      source = "hashicorp/azurerm"
-      version = "~>2.0"
-    }
-  }
-}
-
-provider "azurerm" {
-  features {}
+resource "random_pet" "rg-name" {
```

Comment:
Can we use snake-case here due to the community [convention](https://github.com/terraform-linters/tflint/blob/master/docs/rules/terraform_naming_convention.md) please?

```hcl
resource "random_pet" "rg_name" {
```